common/pkcs11.h

```diff
@@ -403,6 +412,7 @@ typedef unsigned long ck_key_type_t;
 #define CKK_GOSTR3411		(0x31UL)
 #define CKK_GOST28147		(0x32UL)
 #define CKK_EC_EDWARDS		(0x40UL)
+#define CKK_EC_MONTGOMERY	(0x41UL)
```

Comment:
Let's do this in a separate commit.

Reply:
but the commit would consist only of 3 lines, is it even worth splitting?